src/dotnet/Chat.UI.Blazor/Components/MarkupParts/PlainTextPartView.razor

```diff
@@ -8,9 +8,9 @@
 @{
     var textToTimeMap = Part.Markup.TextToTimeMap;
     var isPlayable = Entry.AudioEntryId.HasValue && !textToTimeMap.IsEmpty;
-    var cls = "plain-text-part-view";
+    var cls = string.Empty;
```

Comment:
does `cls` stand for `class` or `css`

Reply:
@FrolYo cls always stands for class.

@evgeniy-ac on a side note, please don't use `string.Empty` - it's a legacy for a while, C# automatically replaces `""` to it anyway.